### DISCUSSION TITLE
web版本 使用canvas.toDataURL生成的是一个空白的图片，怎么可以实现截图呢

### DISCUSSION THREAD

**Discussion by xiaxinran123** [Tencent/libpag#1849] [Q&A] [answered]
使用libpag播放结束后，想将页面进行截图，但是使用canvas.toDataURL 生成的图片是灰色的，没有内容

**Comment by zhalice2011** [comment, marked as answer]
你好, 有提供 makeSnapshot 的 API

在线demo: https://pagtest.zhalice.com/pages/make-snapshot.html
